src/lib/y2network/sysconfig/connection_config_readers/ctc.rb

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "y2network/sysconfig/connection_config_readers/base"
+
+module Y2Network
+  module Sysconfig
+    module ConnectionConfigReaders
+      # This class is able to build a ConnectionConfig::Ctc object given a
+      # Sysconfig::InterfaceFile object.
+      class Ctc < Base
+      private
+
+        # @see Y2Network::Sysconfig::ConnectionConfigReaders::Base#update_connection_config
+        def update_connection_config(_conn)
```

Comment:
this looks strange. Why not having empty method in Base and redefine only when needed?

Reply:
@imobachgs , maybe something you could comment.

Reply:
Yes, fine with having the base method.